### DISCUSSION TITLE
Getting an error Incorrect API key

### DISCUSSION THREAD

**Discussion by srthakkar** [structurizr/lite#85] [Q&A]
Hello,

What is the meaning of this error? We are just running the sample demo app and notice the below error in command line

ERROR 11224 --- [nio-8080-exec-6] com.structurizr.lite.web.ApiController   : com.structurizr.lite.web.HttpUnauthorizedException: Incorrect API key

**Comment by simonbrowndotje** [comment]
Hard to say with the limited information you've provided, but my guess would be another browser tab open with a different Structurizr session.
